js/bybit.js

```diff
@@ -2642,6 +2642,8 @@ module.exports = class bybit extends Exchange {
             'side': side,
             'price': price,
             'stopPrice': stopPrice,
+            'stopLossPrice': this.safeNumber2 (order, 'stop_loss', 'stopLoss'),
```

Comment:
When not set they default to 0 so I guess we should make it undefined in this case

```suggestion
            let stopLossPrice = this.safeNumber2 (order, 'stop_loss', 'stopLoss');
            if (stopLossPrice === 0) {
                stopLossPrice = undefined;
            }
```